### PR TITLE
Fix kubectl in centos default not install bash-completion

### PR DESCRIPTION
**Pull request by chenyb4**
Signed-off-by: Yuanbin.Chen <cybing4@gmail.com>

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.11 Features: set Milestone to 1.11 and Base Branch to release-1.11
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> NOTE: After opening the PR, please *un-check and re-check* the ["Allow edits from maintainers"](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) box so that maintainers can work on your patch and speed up the review process. This is a temporary workaround to address a known issue with GitHub.> 
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

### CODE REVIEW

File: content/en/docs/tasks/tools/install-kubectl.md [hunk 1]
```diff
@@ -217,6 +217,11 @@ The completion script itself is generated by kubectl, so you typically just need
 Common examples are provided here. For more details, consult `kubectl completion -h`.
 
 ### On Linux, using bash
+On CentOS Linux, you will need to install bash-completion support, bash-completion not install in default.
+
```
Comment (tengqm) [inline content/en/docs/tasks/tools/install-kubectl.md]: you may need to install the `bash-completion` package which is not installed by default.

**Comment by fabian4** [comment]
@tengqm 
Done, thanks.

**Pull request merged**